README.md

```diff
@@ -435,24 +435,16 @@ Defaults to `true`, unless the application origin is `http://localhost`, in whic
 
 Defaults to `none` if the `secure` option is `true`, or `lax` if the `secure` option is false. Allows fine-grained control over the same-site cookie setting. A value of `none` allows embedding within an iframe. A value of `lax` will avoid being blocked by user "3rd party" cookie settings. A value of `strict` will block all cookies when redirecting from Okta and is not recommended.
 
-##### `isAuthenticated`
+##### `transformAuthState`
 
-Callback function. By default, the SDK will consider a user authenticated if both valid idToken and accessToken are available from `tokenManager`. Setting a `isAuthenticated` function on the config will skip the default logic and call the supplied function instead. The function should return a Promise and resolve to either true or false. This callback is only evaluated when the `auth` code has reason to think the authentication state has changed, by default it's been triggered when token state changes.
+Callback function. By default, the SDK will consider a user authenticated if both valid idToken and accessToken are available from `tokenManager`. Setting a `transformAuthState` function on the config will emit a new transformed `authState` based on the evaludated authState from the default logic. The function should return a Promise and resolve to an [AuthState](#authstatemanager). This callback is only evaluated when the `auth` code has reason to think the authentication state has changed, by default it's been triggered when token state changes.
```

Comment:
typo: "evaluated"

Reply:
This is a bit confusing: "emit a new transformed `authState` based on the evaludated authState from the default logic."

How about something like: "Callback function. When `updateAuthState` is called a new `authState` object is produced. Providing a `transformAuthState` function allows you to modify or replace this object before it is stored and emitted. A common use case is to change the meaning of `isAuthenticated`. By default, `updateAuthState` will set `isAuthenticated` to **true** if unexpired tokens are available from `tokenManager`. This logic could be customized to also require a valid Okta SSO session:

```
const config = {
  // other config
 transformAuthState: async (oktaAuth, authState) => {
   if (!authState.isAuthenticated) {
     return authState;
  }
  // extra requirement: user must have valid Okta SSO session
  const user = await oktaAuth.token.getUserInfo();
  authState.isAuthenticated = !!user; // convert to boolean
  authState.users = user; // also store user object on authState
  return authState;
};

const oktaAuth = new OktaAuth(config);
oktaAuth.authStateManager.updateAuthState();

```